workflow/applications/applications.py

```diff
@@ -53,6 +53,10 @@ def __init__(self, conf: Configuration) -> None:
         self.do_aero = _base.get('DO_AERO', False)
         self.do_bufrsnd = _base.get('DO_BUFRSND', False)
         self.do_gempak = _base.get('DO_GEMPAK', False)
+        self.do_gempakmeta = _base.get('DO_GEMPAK', False)
+        self.do_gempakmetancdc = _base.get('DO_GEMPAK', False)
+        self.do_gempakncdcupapgif = _base.get('DO_GEMPAK', False)
+        self.do_gempakpgrb2spec = _base.get('DO_GEMPAK', False)
```

Comment:
These are not needed. Only `self.do_gempak` is being used.

Reply:
I added this because my Rocoto mesh wasn't picking up the jobs. Maybe it was a red herring and I will look again but it took me sometime to figure out and why and this seemed to resolve the issue.

Reply:
NVM the above. It must have been a red herring.